### PR TITLE
Websocket Setup

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,12 +1,50 @@
...
+		caller = req.body.Message;
+		res.json({ message: 'Call from ' + caller + ' ended!' });
+		wsc.send("{\"message\" : \"" + caller + "\"}");
+    });
+		
 
 app.listen(4000, function(){
   console.log("Server started at port 4000");
-});
+});
```
Comment (aashutoshrathi) [inline app.js]: New line at the end

File: public/js/call_log.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    try {
+      var json = JSON.parse(message.data);
+	  console.log('JSON: ', json);
+    } catch (e) {
+      console.log('Invalid JSON: ', message.data);
+      return;
+    }
+	document.getElementById('chatlog').innerHTML += '<br>' + json.message;
+	}
+});
```
Comment (aashutoshrathi) [inline public/js/call_log.js]: newline at the end.

File: public/js/call_log.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+]);
+
+$( document ).ready(function() {
+    console.log( "ready!" );
+	window.WebSocket = window.WebSocket || window.MozWebSocket;
+	if (!window.WebSocket) {
+    document.getElementById('chatlog').innerHTML += '<br>Sorry, but your browser doesn\'t support WebSocket.';
+	return;
+    }
+	else
```
Comment (aashutoshrathi) [inline public/js/call_log.js]: Use only one style, as here you are using 
```
if(condition) {
  //do it;
}
```
Follow this everywhere, also fix indentations.

**Comment by jatinluthra14** [comment]
@aashutoshrathi Done and Squashed.

**Comment by KeenWarrior** [comment]
@jatinluthra14 `nohup.out` need to be in commit?

**Comment by jatinluthra14** [comment]
@KeenWarrior no not required, I'll remove it from commit

**Pull request merged**